.rubocop.yml

```diff
@@ -50,12 +54,21 @@ Style/CollectionMethods:
 Style/WordArray:
   Enabled: false
 
+Style/SymbolArray:
+  Enabled: false
+
 Style/RegexpLiteral:
   Enabled: false
 
 Style/StringLiterals:
   Enabled: false
 
+Style/FrozenStringLiteralComment:
+  Enabled: false
```

Comment:
I haven't tried it but setting `TargetRubyVersion` might disable checks that require a newer Ruby http://rubocop.readthedocs.io/en/latest/configuration/#setting-the-target-ruby-version